RELEASE.adoc

```diff
@@ -15,3 +15,11 @@
   git checkout master ## it's only possible to release from master for now!
   git tag release/v1.6.1
   git push git@github.com:jaegertracing/jaeger-operator.git release/v1.6.1
+
+1. Apply generated OLM catalog files to operatorhub.io
+
+* Retrieve the auto-generated operator manifests, e.g. git pull upstream master
+* Clone the operatorhub repo: https://github.com/operator-framework/community-operators
+* Apply the following changes to `community-operators/jaeger` and `upstream-community-operators/jaeger` in separate PRs:
```

Comment:
I think it's worth linking the `upstream-...` repo, as it's the first mention to it.

In a related note: I think we can automate this in our release script, no?

Reply:
Do you fancy giving it a go before considering merging this?

Although you can push a branch, you will still need someone to go onto github to create the PR? Or is there a way to automate that?

Reply:
We just need to send the PR, right? I think we can do that with the same account we use for pushing the tags. 

PRs can be created via REST, but it would probably be easier to use [`hub`](https://hub.github.com/) for that.

Reply:
About giving this a shot: I could, just not sure what are the current priorities.